src/AccessibilityInsights.AutomationTests/AutomationSessionUnitTests.cs

```diff
@@ -57,22 +62,19 @@ public void NewInstance_NoInstanceExists_ExceptionInCtor_RethrowsSameException()
                 }
             }
         }
+#endif
 
         [TestMethod]
         [Timeout (2000)]
         public void NewInstance_ClearInstance_NotPowerShell_AssemblyResolverIsNull_DoesNotThrow()
```

Comment:
>NewInstance_ClearInstance_NotPowerShell_AssemblyResolverIsNull_DoesNotThrow [](start = 20, length = 75)

I want to double-check the internals to confirm that these shims aren't needed internally to exercise the proper code.